packages/node/src/index.ts

```diff
@@ -1,3 +1,5 @@
-import Node from "./node";
+import { Node, NodeOptions } from "./node";
 
-export { Node };
+export function create(options: NodeOptions) {
```

Comment:
create wrappers around `new` keyword usually a bad idea

Reply:
Doing `new cfNode.Node()` is rather awkward tbh

Reply:
Also why prefix with `cf`? We internally call this thing `Node` but thats just a super vanilla name, we can come up with something better